kie-server-parent/kie-server-services/kie-server-services-common/src/test/java/org/kie/server/services/impl/security/JACCIdentityProviderWildFlyTomcatTest.java

```diff
@@ -101,6 +102,9 @@ public void testGetRoles() throws Exception {
         assertEquals(2, jaccIdentityProvider.getRoles().size());
         assertTrue(jaccIdentityProvider.getRoles().contains(GROUP_ONE_NAME));
         assertTrue(jaccIdentityProvider.getRoles().contains(GROUP_TWO_NAME));
+        assertTrue(jaccIdentityProvider.hasRole(GROUP_ONE_NAME));
+        assertTrue(jaccIdentityProvider.hasRole(GROUP_TWO_NAME));
+        assertFalse(jaccIdentityProvider.hasRole("non_existing_role"));
```

Comment:
You can add a test for null role:
`assertFalse(jaccIdentityProvider.hasRole(null));`